usaspending_api/agencies/management/commands/load_agencies.py

```diff
@@ -0,0 +1,248 @@
+import logging
+
+from collections import namedtuple
+from distutils.util import strtobool
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from pathlib import Path
+from psycopg2.extras import execute_values
+from usaspending_api.common.csv_helpers import read_csv_file_as_list_of_dictionaries
+from usaspending_api.common.helpers.sql_helpers import get_connection
+from usaspending_api.common.helpers.text_helpers import standardize_nullable_whitespace as prep
+from usaspending_api.common.helpers.timing_helpers import Timer
+
+
+logger = logging.getLogger("console")
+
+
+Agency = namedtuple(
+    "Agency",
+    [
+        "row_number",
+        "cgac_agency_code",
+        "agency_name",
+        "agency_abbreviation",
+        "frec",
+        "frec_entity_description",
+        "frec_abbreviation",
+        "subtier_code",
+        "subtier_name",
+        "subtier_abbreviation",
+        "toptier_flag",
+        "is_frec",
+        "user_selectable",
+        "mission",
+        "website",
+        "congressional_justification",
+        "icon_filename",
+    ],
+)
+
+
+# Set keep_count to True to update the overall affected row count.  Set skip_if_no_changes to True to not run
+# this step if the overall affected row count is zero.  Useful for not running expensive steps if nothing changed.
+ProcessingStep = namedtuple("ProcessingStep", ["description", "file", "function", "keep_count", "skip_if_no_changes"])
```

Comment:
nice

Reply:
Is there no way to have the function be an actual function as a first-class citizen instead of a string you have to use reflection on?

Reply:
There is, and I really wanted to do that, but it made the code a bit messier.  I will take another look at that and see if I can find a cleaner way.  I especially don't like having both a file and a function.  It's only ever going to be one or the other.